dist/localization.json

```
@@ -86,9 +86,9 @@
     "none": "none",
     "warns_count": "Total Count: {0} Warns",
     "warn_remove_error_null_or_negative": "WarnId {0} cannot be negative or null.",
-    "warn_remove_error_notExistant": "The Warn {0} does not exist",
+    "warn_remove_error_notExistent": "The Warn {0} does not exist",
```

Follow one format, change camelCase to snake_case.
```suggestion
    "warn_remove_error_not_existent": "The Warn {0} does not exist",
```